doc/src/guide/listeners.asciidoc

```diff
@@ -203,6 +203,23 @@ ranch:set_max_connections(tcp_echo, MaxConns).
 
 The change will occur immediately.
 
+=== Running out of file descriptors
```

Comment:
I'm not sure this title is good. We don't want to run out of file descriptors, it's not something we want to do.

Perhaps something like "When running out of file descriptors" would be a little more clear, though I'm open to suggestions.